dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationResourceTest.java

```diff
@@ -0,0 +1,170 @@
+        Assert.assertNotNull(serviceIntegrationResponse);
+        Assert.assertEquals(HttpStatus.SC_OK, serviceIntegrationResponse.getStatus());
+        final Response availableServicesResponse = serviceIntegrationResource.listAvailableServices(request, response);
+        Assert.assertEquals(HttpStatus.SC_OK, availableServicesResponse.getStatus());
+        final ResponseEntityView responseEntityView1 = (ResponseEntityView)availableServicesResponse.getEntity();
+        final List<ServiceIntegrationView> integrationViewList = (List<ServiceIntegrationView>)responseEntityView1.getEntity();
+        Assert.assertFalse(integrationViewList.isEmpty());
+        Assert.assertTrue(
+                integrationViewList.stream().anyMatch(serviceIntegrationView ->  "lola".equals(serviceIntegrationView.getName())));
+
+        final SecretForm secretForm = new SecretForm();
+        secretForm.setServiceKey(serviceKey);
+        secretForm.setHostId(host.getIdentifier());
+        secretForm.setParams(paramMap);
+        final Response createSecretResponse = serviceIntegrationResource.createSecret(request, response, secretForm);
+        Assert.assertEquals(HttpStatus.SC_OK, createSecretResponse.getStatus());
+
+        final Response hostIntegrationsResponse = serviceIntegrationResource.getServiceIntegrationByKey(request, response, serviceKey);
+        Assert.assertEquals(HttpStatus.SC_OK, hostIntegrationsResponse.getStatus());
+        final ResponseEntityView responseEntityView2 = (ResponseEntityView) hostIntegrationsResponse.getEntity();
+        final ServiceIntegrationHostView serviceIntegrationHostView2 = (ServiceIntegrationHostView) responseEntityView2.getEntity();
+        final ServiceIntegrationView serviceIntegrationView2 = serviceIntegrationHostView2.getService();
+        Assert.assertEquals(1, serviceIntegrationView2.getConfigurationsCount());
+        Assert.assertEquals("lola", serviceIntegrationView2.getName());
+        final List<HostView> hosts = serviceIntegrationHostView2.getHosts();
+        Assert.assertNotNull(hosts);
+        Assert.assertFalse(hosts.isEmpty());
+        Assert.assertEquals(hosts.get(0).getHostId(),host.getIdentifier());
+        Assert.assertTrue(
+                hosts.stream().anyMatch(hostView -> host.getIdentifier().equals(hostView.getHostId()))
+                );
+
+        final Response detailedIntegrationResponse = serviceIntegrationResource.getDetailedServiceIntegration(request, response, serviceKey, host.getIdentifier());
+        Assert.assertEquals(HttpStatus.SC_OK, detailedIntegrationResponse.getStatus());
+        final ResponseEntityView responseEntityView3 = (ResponseEntityView) detailedIntegrationResponse.getEntity();
+        final ServiceIntegrationDetailedView serviceIntegrationDetailedView = (ServiceIntegrationDetailedView) responseEntityView3.getEntity();
+        System.out.println(serviceIntegrationDetailedView.getHost());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)